src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineFieldDescriptor.php

```diff
@@ -117,7 +117,7 @@ public function getJoins()
     /**
      * {@inheritdoc}
      */
-    public function compare(FieldDescriptorInterface $other)
+    public function compare(self $other)
```

Comment:
Did get a interface error in the IDE should we set it to the FieldDescriptorInterface or let it on self